static/js/services/contact-view-model-generator.js

```diff
@@ -147,7 +147,25 @@ angular.module('inboxServices').factory('ContactViewModelGenerator',
       return DB().query('medic-client/contacts_by_parent', {
         key: contactId,
         include_docs: true
-      });
+      })
+        .then(function(childrenResponse) {
+          var ids = childrenResponse.rows.map(function(child) {
+            return child.doc.contact && child.doc.contact._id;
+          });
+          return DB().query('medic-client/doc_summaries_by_id', { keys: ids })
+            .then(function(contactsResponse) {
+              childrenResponse.rows.forEach(function(row) {
+                var contactId = row.doc.contact && row.doc.contact._id;
+                if (contactId) {
+                  var contactRow = _.findWhere(contactsResponse.rows, { id: contactId });
+                  if (contactRow) {
+                    row.doc.contact.name = contactRow.value.name;
```

Comment:
That looks unexpected to me, I'd expect either no hydration or full hydration. I wouldn't be surprised if it caused bugs later. 
Also, do we know why only the contact names are needed? Or are we going to be adding phone numbers next week, and then grandparent names the week after, and then... 
Thoughts?